firmware/application/event_m0.cpp

```diff
@@ -281,13 +281,15 @@ void EventDispatcher::emulateTouch(ui::TouchEvent event) {
     while (injected_touch_event != nullptr) {
         chThdSleepMilliseconds(5);
     }
+    injected_touch_event = nullptr;  // to clean event_mo.cpp, compile warning error : "storing the address of local variable 'event' in 'this_4(D)->injected_touch_event' [-Wdangling-pointer=]"
```

Comment:
Looks like this should work.  Another option would be to disable the bogus warning using something like this around line 280, but it looks more messy than your 1-line change:

```
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wdangling-pointer"
    injected_touch_event = &event;    // the original line causing the warning
#pragma GCC diagnostic pop
```

Reply:
We will keep his version then.
Thanks for the info, I remember having to play with pragmas, fun times ;-)